metadata-models/src/main/pegasus/com/linkedin/ml/metadata/MLFeatureProperties.pdl

```diff
@@ -25,4 +26,15 @@ record MLFeatureProperties {
   * Version of the MLFeature
   */
   version: optional VersionTag
+
+  /**
+  * Source of the MLFeature
+  */
+  @Relationship = {
+    "/*": {
+      "name": "DerivedFrom",
+      "entityTypes": [ "dataset" ]
+    }
+  }
+  sources: optional array[Urn]
```

Comment:
Should this be a separate "lineage" aspect? Will we ever need to update this field independently of the others?